### PR TITLE
🏗 Stop running visual diff tests on Travis

### PR DESCRIPTION
**Pull request by rsimha**
They're already being run on CircleCI, and we're shutting off Travis soon.

### CODE REVIEW

File: build-system/pr-check/visual-diff-tests.js [hunk 1]
```diff
@@ -52,4 +53,12 @@ function prBuildWorkflow() {
   }
 }
 
+// TODO(rsimha): Remove this block once Travis is shut off.
+if (isTravisBuild()) {
+  printSkipMessage(
+    jobName,
+    'this is a Travis build. Sizes will be reported from CircleCI'
```
Comment (danielrozenberg) [inline build-system/pr-check/visual-diff-tests.js:60]: Copy and pasted text, I assume?

**Review by danielrozenberg — APPROVED** [review]
There's an explicit check for Travis PRs in `tasks/visual-diff/index.js`, should change that too

File: build-system/pr-check/visual-diff-tests.js (hunk 1 above)
Comment (rsimha) [reply build-system/pr-check/visual-diff-tests.js:60]: Hah! Yes. Fixed.

**Comment by rsimha** [comment]
This worked: https://travis-ci.com/github/ampproject/amphtml/jobs/481651644

**Pull request merged**